ush/forecast_postdet.sh

```diff
@@ -0,0 +1,959 @@
+    cd $SCRIPTDIR
+    echo "MAIN: !!!Sandbox mode, writing to current directory!!!"
+  fi
+  # Call child scripts in current script directory
+  source $SCRIPTDIR/parsing_namelists_FV3.sh
+  FV3_namelists
+  echo SUB ${FUNCNAME[0]}: FV3 name lists and model configure file created
+}
+
+DATM_nml(){
+  source $SCRIPTDIR/parsing_namelists_DATM.sh
+  DATM_namelists
+  echo SUB ${FUNCNAME[0]}: DATM name lists and model configure file created
+}
+
+data_out_GFS() {
+  # data in take for FV3GFS
+  # Arguments: None 
+  # 
+  #------------------------------------------------------------------
+  # make symbolic links to write forecast files directly in memdir
+  echo "SUB ${FUNCNAME[0]}: copying output data for FV3"
+  #------------------------------------------------------------------
+  
+  if [ $SEND = "YES" ]; then
+    # Copy model restart files
+    if [ $CDUMP = "gdas" -a $restart_interval -gt 0 ]; then
+      cd $DATA/RESTART
+      mkdir -p $memdir/RESTART
+      # Only save restarts at single time in RESTART directory
+      # Either at restart_interval or at end of the forecast
+      #  if [ $restart_interval -eq 0 -o $restart_interval -eq $FHMAX ]; then
+      
+      # Add time-stamp to restart files at FHMAX
+      #    RDATE=$($NDATE +$FHMAX $CDATE)
+      #    rPDY=$(echo $RDATE | cut -c1-8)
+      #    rcyc=$(echo $RDATE | cut -c9-10)
+      #    for file in $(ls * | grep -v 0000); do
+      #      $NMV $file ${rPDY}.${rcyc}0000.$file
+      #    done
+      #  else
+      # time-stamp exists at restart_interval time, just copy
+      RDATE=$($NDATE +$restart_interval $CDATE)
+      rPDY=$(echo $RDATE | cut -c1-8)
+      rcyc=$(echo $RDATE | cut -c9-10)
+      for file in ${rPDY}.${rcyc}0000.* ; do
+      	$NCP $file $memdir/RESTART/$file
+      done
+    fi
+  fi
+  
+  $NCP $DATA/input.nml $ROTDIR/${CDUMP}.${PDY}/${cyc}/atmos/
+  
+  echo "SUB ${FUNCNAME[0]}: Output data for FV3 copied"
+}
+
+WW3_postdet() {
+  echo "SUB ${FUNCNAME[0]}: Linking input data for WW3"
+  COMPONENTwave=${COMPONENTwave:-${RUN}wave}
+  
+  #Link mod_def files for wave grids
+  array=($WAVECUR_FID $WAVEICE_FID $WAVEWND_FID $waveuoutpGRD $waveGRD $waveesmfGRD $wavesbsGRD $wavepostGRD $waveinterpGRD)
+  echo "Wave Grids: $WAVECUR_FID $WAVEICE_FID $WAVEWND_FID $waveuoutpGRD $waveGRD $waveesmfGRD $wavesbsGRD $wavepostGRD $waveinterpGRD"
+  grdALL=`printf "%s\n" "${array[@]}" | sort -u | tr '\n' ' '`
+  
+  for wavGRD in ${grdALL}; do
+  	$NCP $ROTDIR/${CDUMP}.${PDY}/${cyc}/wave/rundata/${COMPONENTwave}.mod_def.$wavGRD $DATA/mod_def.$wavGRD
+  done
+  
+  export WAVHCYC=${WAVHCYC:-6}
+  export WRDATE=`$NDATE -${WAVHCYC} $CDATE`
+  export WRPDY=`echo $WRDATE | cut -c1-8`
+  export WRcyc=`echo $WRDATE | cut -c9-10`
+  export WRDIR=${ROTDIR}/${CDUMPRSTwave}.${WRPDY}/${WRcyc}/wave/restart
+  export datwave=$COMOUTwave/rundata
+  export wavprfx=${CDUMPwave}${WAV_MEMBER}
+  
+  #Copy initial condition files:
+  for wavGRD in $waveGRD ; do
```

Comment:
The wave parts in this next section all appear to have been changed from what's in f/c-c to match what is in develop branch.  Just making a note that when we need to reconcile with reproducing f/c-c the differences here will have to be dealt with.  Nothing to change right now though.